<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintPanel</class>
 <widget class="QDockWidget" name="PrintPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>599</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>599</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>388</width>
    <height>524287</height>
   </size>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Print Scene</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageBeforePrint">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QComboBox" name="toolCombo">
          <property name="toolTip">
           <string>Pick a tool-script with which to print</string>
          </property>
          <property name="statusTip">
           <string>Pick a tool-script with which to print</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="textToolScriptDescription">
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>To use this script, you need:

    - Two bays available on the Fab@Home Model 2
    - A 10 cc syringe with an orange (1 mm) tip
    - A 10 cc syringe with a blue (0.5mm) tip
    - FabEpoxy, 3:1 mixture
    - Frosting

Be sure to have extra supplies on hand if you are printing a large model.  This script will automatically support the object you are printing so it won't fall over, even if it has overhangs.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="print">
          <property name="text">
           <string>Create Print Job</string>
          </property>
          <property name="description">
           <string>Generates instructions for the printer .  This can take several minutes.  You can review the results before the printer starts building.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="modifyPrintSettings">
          <property name="text">
           <string>Modify Tool and Printer Settings (Advanced)</string>
          </property>
          <property name="description">
           <string>Edit the parameter values used by the print script, choose alternate planning algorithms, or give FabStudio hints about how to create your print job.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageDuringPrint">
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QGroupBox" name="progressGroupBox">
          <property name="title">
           <string>Working....</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <widget class="QProgressBar" name="taskProgressBar">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="masterProgressBar">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>48</height>
              </size>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="sendToPrinter">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Send to Printer</string>
          </property>
          <property name="description">
           <string>Connect to the printer and send it the current scene.  The program will step you through attaching the necessary tools.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="savePrintJob">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save to a File (Advanced)</string>
          </property>
          <property name="description">
           <string>Store this print job in a .FAB file.  This will save only the information the printer needs and not the original objects.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="abandonPrintJob">
          <property name="text">
           <string>Start Over</string>
          </property>
          <property name="description">
           <string>Discard this print job and go back to loading objects and assigning materials.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
